pkg/cmd/cmd.go

```diff
@@ -98,6 +100,7 @@ func NewGardenctlCommand(f *util.FactoryImpl, ioStreams util.IOStreams) *cobra.C
 	cmd.AddCommand(cmdssh.NewCmdSSH(f, cmdssh.NewSSHOptions(ioStreams)))
 	cmd.AddCommand(cmdtarget.NewCmdTarget(f, cmdtarget.NewTargetOptions(ioStreams)))
 	cmd.AddCommand(cmdversion.NewCmdVersion(f, cmdversion.NewVersionOptions(ioStreams)))
+	cmd.AddCommand(cmdConfig.NewCmdConfig(f, cmdConfig.NewConfigOptions(ioStreams)))
```

Comment:
new commands should pass the `ioStreams` and the options should be created internally.